q2_moshpit/_utils.py

```diff
@@ -100,3 +103,62 @@ def get_feature_lengths(features: MAGSequencesDirFmt) -> pd.DataFrame:
     df = pd.DataFrame({'id': ids, 'length': lengths})
     df.set_index('id', inplace=True)
     return df
+
+
+def _multiply(
+        table1: pd.DataFrame, table2: pd.DataFrame
+) -> pd.DataFrame:
+    """Calculate dot product of two tables."""
+    # if table1.shape[1] != table2.shape[0]:
+    #     raise ValueError(
+    #         f"Tables do not have compatible dimensions for dot product: "
+    #         f"{table1.shape[1]} != {table2.shape[0]}."
+    #     )
```

Comment:
Do we need this comment?

Reply:
Oh, no, just some left-over - I'm removing it.